apps/src/aichat/views/ModelCustomizationWorkspace.tsx

```diff
@@ -107,6 +107,10 @@ const ModelCustomizationWorkspace: React.FunctionComponent = () => {
     [setSelectedTab]
   );
 
+  if (visibleTabs.length === 0) {
```

Comment:
Unfortunately I can't put it above the `onChange` useCallback since hooks can't be called conditionally (can't early return before a hook). I could move the onChange up too, but I also think it makes sense to keep it close to the render where it's used? Will add a comment though